modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1989,6 +1989,8 @@
     <string name="clear_before_adding">Clear Before Adding</string>
     <string name="dont_clear">Don\'t Clear</string>
     <string name="clear_up_next">Clear Up Next</string>
+    <string name="up_next_shuffle_button_content_description">Up next shuffle button</string>
+    <string name="up_next_shuffle_disable_button_content_description">Up next shuffle disable button</string>
```

Comment:
```suggestion
    <string name="up_next_shuffle_button_content_description">Up next shuffle</string>
    <string name="up_next_shuffle_disable_button_content_description">Up next shuffle disable</string>
```
In TalkBack this reads as "Up next shuffle button. Button".

<img  width="300" src="https://github.com/user-attachments/assets/2316fccc-ea4f-4def-b35c-77f4c48ae3dd" />